layers/+misc/multiple-cursors/README.org

```diff
@@ -20,8 +20,7 @@ add =multiple-cursors= to the existing =dotspacemacs-configuration-layers= list
 file.
 
 * Configuration
-Currently the only supported backend is =evil-mc=, but more backends will be
-available in the future.
+Currently supported backends are =evil-mc= (default) and =mc=.
```

Comment:
Maybe the text `evil-mc` and `mc` could link to the packages:
https://github.com/gabesoft/evil-mc
https://github.com/magnars/multiple-cursors.el

Should we add parentheses after `mc`  to indicate that it stands for `(multiple cursors)` or possibly even 
`(magnars multiple-cursors)` so as not to confuse it with the layer name.

Reply:
I think links are enough.